handlers/mempoolView.go

```diff
@@ -54,6 +54,32 @@ func formatToTable(content *types.RawMempoolResponse) *types.DataTableResponse {
 			})
 		}
 	}
+	for _, pendingData := range content.BaseFee {
+		for _, tx := range pendingData {
+			dataTable.Data = append(dataTable.Data, []any{
+				utils.FormatAddressWithLimits(tx.Hash.Bytes(), "", false, "tx", 15, 18, true),
+				utils.FormatAddressAll(tx.From.Bytes(), "", false, "address", "", int(12), int(12), true),
+				_isContractCreation(tx.To),
+				utils.FormatAmount((*big.Int)(tx.Value), "ETH", 5),
+				utils.FormatAddCommasFormated(float64(tx.Gas.ToInt().Int64()), 0),
+				utils.FormatAmountFormated(tx.GasPrice.ToInt(), "GWei", 5, 0, true, true, false),
+				tx.Nonce.ToInt(),
+			})
+		}
+	}
+	for _, pendingData := range content.Queued {
+		for _, tx := range pendingData {
+			dataTable.Data = append(dataTable.Data, []any{
+				utils.FormatAddressWithLimits(tx.Hash.Bytes(), "", false, "tx", 15, 18, true),
+				utils.FormatAddressAll(tx.From.Bytes(), "", false, "address", "", int(12), int(12), true),
+				_isContractCreation(tx.To),
+				utils.FormatAmount((*big.Int)(tx.Value), "ETH", 5),
+				utils.FormatAddCommasFormated(float64(tx.Gas.ToInt().Int64()), 0),
+				utils.FormatAmountFormated(tx.GasPrice.ToInt(), "GWei", 5, 0, true, true, false),
+				tx.Nonce.ToInt(),
+			})
+		}
+	}
```

Comment:
Lots of duplicated code, consider doing something like this:
```
func formatToTable(content *types.RawMempoolResponse) *types.DataTableResponse {
	dataTable := &types.DataTableResponse{}

	for _, pendingData := range content.Pending {
		for _, tx := range pendingData {
			dataTable.Data = append(dataTable.Data, txToInterfaces(tx))
		}
	}
	for _, pendingData := range content.BaseFee {
		for _, tx := range pendingData {
			dataTable.Data = append(dataTable.Data, txToInterfaces(tx))
		}
	}
	for _, pendingData := range content.Queued {
		for _, tx := range pendingData {
			dataTable.Data = append(dataTable.Data, txToInterfaces(tx))
		}
	}
	return dataTable
}

func txToInterfaces(tx *types.RawMempoolTransaction) []interface{} {
	return []interface{}{
		utils.FormatAddressWithLimits(tx.Hash.Bytes(), "", false, "tx", 15, 18, true),
		utils.FormatAddressAll(tx.From.Bytes(), "", false, "address", "", int(12), int(12), true),
		_isContractCreation(tx.To),
		utils.FormatAmount((*big.Int)(tx.Value), "ETH", 5),
		utils.FormatAddCommasFormated(float64(tx.Gas.ToInt().Int64()), 0),
		utils.FormatAmountFormated(tx.GasPrice.ToInt(), "GWei", 5, 0, true, true, false),
		tx.Nonce.ToInt(),
	}
}
```

If you do so, `txToInterfaces` should get a better name though.

Reply:
done